app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -95,6 +95,24 @@ def usps_proofer
         end
       end
 
+      def localized_hours(hours)
+        if hours == 'Closed'
+          I18n.t('in_person_proofing.body.barcode.retail_hours_closed')
+        elsif hours.include?(' - ') # Hyphen
+          hours.
+            split(' - '). # Hyphen
+            map { |time| Time.zone.parse(time).strftime(I18n.t('time.formats.event_time')) }.
```

Comment:
What about `I18n.l`?

```suggestion
            map { |time| I18n.l(Time.zone.parse(time), format: :event_time) }.
```

Reply:
This surprisingly changes the format of the string, so I've reverted it, but we should probably investigate that.